crates/iota-indexer/src/backfill/ingestion/jobs/mod.rs

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Mysten Labs, Inc.
+// Modifications Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Nit: from my understanding we could remove the modifications part since is is a new file not present in the upstream and the code does target different use case?

```suggestion
// Copyright (c) 2025 IOTA Stiftung
// SPDX-License-Identifier: Apache-2.0
```

Reply:
Yes, I think it we can change it! Our implementation is original enough. Thanks @sergiupopescu199!